index.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>We shouldn't limit this to 10 numbers</p>

 _Reply via <a href='https://app.reviewnb.com/osule/explore-reviewnb/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>